bundles/org.openhab.binding.enturno/src/main/java/org/openhab/binding/enturno/internal/connection/EnturConfigurationException.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.enturno.internal.connection;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link EnturConfigurationException} is a configuration exception for the connections to Entur API.
+ *
+ * @author Michal Kloc - Initial contribution
+ */
+@NonNullByDefault
+public class EnturConfigurationException extends IllegalArgumentException {
+
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * Constructs a new exception with null as its detail message.
+     */
```

Comment:
Please remove javadoc of methods this class (inherit javadoc instead)

Reply:
Hi David, do you mean putting @inheritDoc on class level, putting it on all methods in this class or just doing nothing and documentation will be inherited?

```suggestion
    /**
     * @inheritDoc
     */
```

Reply:
These are all constructors so the javadoc you created is fine and should not be changed. @inheritdoc doesn't apply for constructors.

Reply:
But the javadoc is pretty much useless. And we agreed of not wanting useless javadoc

Reply:
Done